### PR TITLE
Added proposal to change voting period

### PR DESCRIPTION
**Pull request by blahkheart**
# Description

This proposal sets the voting delay, and voting period in the Governor contract, as well as the minimum delay for execution (minDelay) in the Timelock contract.

# Issues

Fixes #
Refs #

# Checklist:

- [x ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by blahkheart** [comment]
@julien51 @clemsos please review

**Review by pi0neerpat — CHANGES_REQUESTED** [review]
The function signatures for governance are different now that we are on Base. See the new function signatures in https://github.com/unlock-protocol/unlock/pull/14849. 

I updated my PR there, which could replace this PR (aside from the readme changes)

File: governance/README.md
Comment (clemsos) [inline governance/README.md:1]: Thanks for updating the docs !

File: governance/proposals/up/002-change-voting-duration.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+const { ethers } = require('hardhat')
+const { base } = require('@unlock-protocol/networks')
+const { getNetwork } = require('@unlock-protocol/hardhat-helpers')
+
+const VOTING_DELAY = 4 * 24 * 60 * 60 // 4 days in seconds
+const VOTING_PERIOD = 6 * 24 * 60 * 60 // 6 days in seconds
+const MIN_DELAY = 2 * 24 * 60 * 60 // 2 days in seconds
+const BASE_TIMELOCK_ADDRESS = '0xB34567C4cA697b39F72e1a8478f285329A98ed1b'
+const BASE_GOVERNOR_ADDRESS = '0x65bA0624403Fc5Ca2b20479e9F626eD4D78E0aD9'
```
Comment (clemsos) [inline governance/proposals/up/002-change-voting-duration.js:9]: There is now a better way to get the gov address - from the dao section in the networks package.

File: governance/proposals/up/002-change-voting-duration.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+This proposal sets the voting delay, and voting period in
+the Governor contract, as well as the minimum delay for
+execution (minDelay) in the timelock contract.  
+
+## About this proposal
+
+The proposal contains calls to update the voting delay, voting period, and minDelay using the respective functions.`
+
+  console.log(proposalName)
+  console.log(calls)
```
Comment (clemsos) [inline governance/proposals/up/002-change-voting-duration.js:73]: Please clean these up

**Review by clemsos — COMMENTED** [review]
Lgtm so far. There has been some changes in the gov abi. You can refer to the other PR by @pi0neerpat 

Also it appears this is duplicate with #14849 so maybe that PR can be merged into the other one?

File: governance/proposals/up/002-change-voting-duration.js (hunk 1 above)
Comment (blahkheart) [reply governance/proposals/up/002-change-voting-duration.js:9]: Please show me the better way to get the governor address you are referring to, maybe I'm missing something but I couldn't the governor address in the networks package, I only found an UnlockAddress property.

Comment (blahkheart) [reply governance/proposals/up/002-change-voting-duration.js:9]: local-unlock-networks-log  

This is what it looks like when I log to the console.  I pulled latest changes, deleted node modules and cleaned the cache but still get the same thing, no dao section available.  

unlock-networks-package  

However, if I go to the folder [unlock/packages/networks/src/networks/base.ts](https://github.com/unlock-protocol/unlock/blob/5580b88715c348444a9a6586d28c570a15f3f4fd/packages/networks/src/networks/base.ts#L18) after pulling the latest changes I see the dao section there. Not sure why that's happening.  
This is how I import the package:  
```
const {base }= require('@unlock-protocol/networks')

```

**Comment by blahkheart** [comment]
@clemsos @julien51 The proposal simulation is passing now, I upgraded to node 22.11.0, deleting the node modules and clearing the cache.

File: governance/proposals/up/002-change-voting-duration.js (hunk 1 above)
Comment (clemsos) [reply governance/proposals/up/002-change-voting-duration.js:9]: you may need to run `yarn && yarn build` from the root of the monorepo so the packages are rebuilt

**Comment by blahkheart** [comment]
@clemsos implemented changes and simulation passing, please review

**Review by julien51 — COMMENTED** [review]
LGTM, let's give @clemsos time for a final check and then let's merge and submit onchain please!

**Pull request merged**